### DISCUSSION TITLE
Do you plan to Release GingerXR for xreal Beam?

### DISCUSSION THREAD

**Discussion by ak3dkung** [GingerXR/products#6] [Ideas]
I think it is the good idea if you release it

**Comment by GingerXR** [comment]
would love to. I don't think Xreal updated their SDK to support beam yet.

**Reply by ak3dkung** [reply]
I don't know in term of engineering....but If only made a multiple display in GingerXR for PC and cast it via miracast I think it enough for using Spatial Display on xreal Beam (let's beam doing the head tracking work)
.
Just thinking this way

**Reply by GingerXR** [reply]
I don't know if that will work. GingerXR allows you to create the virtual monitors without being connected right now. if you can cast all those display onto the beam, then no work is even needed?
